portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,149 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+let eerieModeOn = false; 
+
+const root = document.documentElement;
+
+const secretText = document.querySelector("#secret-text"); 
+const mePic = document.querySelector("#me-pic"); 
+const tomiePic = document.querySelector("#tomie-pic"); 
+const mainHeading = document.querySelector("#main-heading"); 
+
+//fade in main text
+const mainText = document.querySelector(".main-text");
+window.addEventListener('load', e => {
+    mainText.classList.add("fade-text-in"); 
+    extraContent.style.display = "none"; 
+})
+
 /**
- * Adds a random greeting to the page.
+ * Keeps track of mouse and updates our special, circle pointer to
+ * follow cursor movement. 
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+const circleAroundMouse = document.querySelector(".cursor");
+window.addEventListener('mousemove', e => {
+    circleAroundMouse.style.top = e.pageY + "px"; 
+    circleAroundMouse.style.left = e.pageX + "px"; 
+})
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+const secretMessages = ["...", "....", "Would you like to see something fun?"]; 
```

Comment:
for variables like this that are the equivalent of static final in Java (true constants, it's annoying that JS overloads the term "constant") declare them at the top in all caps.